app/src/main/res/layout/return_to_topic_button_item.xml

```diff
@@ -14,9 +14,8 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:gravity="center_vertical"
-    app:layoutMarginEnd="@{buttonViewModel.hasConversationView ? @dimen/general_button_item_exploration_view_margin_end : @dimen/general_button_item_question_view_margin_end}"
-    app:layoutMarginStart="@{buttonViewModel.hasConversationView ? @dimen/general_button_item_exploration_view_margin_start : @dimen/general_button_item_question_view_margin_start}"
+    android:paddingEnd="@{buttonViewModel.hasConversationView ? @dimen/general_button_item_exploration_view_margin_end : @dimen/general_button_item_question_view_margin_end}"
+    android:paddingStart="@{buttonViewModel.hasConversationView ? @dimen/general_button_item_exploration_view_margin_start : @dimen/general_button_item_question_view_margin_start}"
```

Comment:
I think you will need to apply this to other buttons too like `continue_button`, `previous_button`, `next_button` etc. 
I suggest fixing all those buttons in single PR as you will need to rename the dimens too considering these are not margins anymore, instead they are padding.

`general_button_item_exploration_view_margin_end ` -> `general_button_item_exploration_view_padding_end `

Reply:
Added padding dimensions in dimens file. `continue_button`, `previous_button`, `next_button`, and `submit_button` are correct in RTL. I have added screenshots in description for reference.